internal/providers/terraform/parser.go

```diff
@@ -100,7 +100,16 @@ func (p *Parser) parseJSONResources(parsePrior bool, baseResources []*schema.Res
 	p.stripDataResources(resData)
 
 	for _, d := range resData {
-		if r := p.createResource(d, usage[d.Address]); r != nil {
+		var usageData *schema.UsageData
+
+		if ud := usage[d.Address]; ud != nil {
+			usageData = ud
+		} else if strings.HasSuffix(d.Address, "]") {
+			if arrayUsageData := usage[fmt.Sprintf("%s*]", d.Address[:len(d.Address)-2])]; arrayUsageData != nil {
```

Comment:
Does this work when count >= 10?

Reply:
Do you mean 'count' as lenght of d.Address? If yes, sure. I checked it in situations with arrays of modules, it was over ~20-30 symbols and it works fine.

Reply:
I mean where `count = 11` in the resource. So the resource would have `aws_instance.my_instance[10]` as the address.

Reply:
ohh, missunderstanding, forgot about that. I will add checking an array lenght before it.

Reply:
Fixed! Ready to review